mklove/modules/configure.cc

```diff
@@ -169,7 +167,7 @@ for n in CFLAGS CPPFLAGS CXXFLAGS LDFLAGS ARFLAGS; do
     mkl_option "Compiler" "mk:$n" "--$n=$n" "Add $n flags"
 done
 
-mkl_option "Compiler" "env:PKG_CONFIG_PATH" "--pkg-config-path" "Extra paths for pkg-config"
+mkl_option "Compiler" "env:PKG_CONFIG_PATH" "--pkg-config-path=PKG_CONFIG_PATH" "Extra paths for pkg-config" "\$PKG_CONFIG_PATH"
```

Comment:
This will overwrite the env PKG_CONFIG_PATH, while the help text states "Extra paths", i.e., additional paths.

I suggest keeping the removed line at 108, but using an internal variable "append_PKG_CONFIG_PATH" that we set at line 172 instead of PKG_CONFIG_PATH.